src/components/ColumnHeaderToolTip/ColumnHeaderToolTip.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import theme from '../../theme'
+
+export const TooltipPopup = styled('span')`
+  display: block;
+  max-width: 25rem;
+  width: max-content;
+  background: ${theme.color.primaryColor};
+  color: ${theme.color.white};
+  position: absolute;
+  font-size: ${theme.typography.smallFontSize};
+  clip-path: polygon(0% 0%, 100% 0%, 100% 83%, 53% 83%, 47% 100%, 40% 83%, 0 83%);
+  padding: 1em;
+  padding-bottom: ${(props) => props.paddingBottom || `calc(4rem - 15px)`};
+  bottom: ${(props) => props.bottom || '4em'};
+  left: ${(props) => props.left || '0'};
+  white-space: normal;
+  z-index: 100;
+  text-align: left;
+`
+
+const ColumnHeaderToolTip = ({ helperText, bottom, left, paddingBottom }) => {
+  return (
+    <TooltipPopup
+      role="tooltip"
+      aria-labelledby="tooltip"
+      bottom={bottom}
```

Comment:
needed to make custom adjustments depending on which table the info icon is being used in (wanted to avoid clipping depending on text length, window size, etc)